src/components/ItemRecordDetailPane/ItemRecordDetailPane.tsx

```diff
@@ -0,0 +1,162 @@
+import React from 'react';
+import { FormattedMessage } from 'react-intl';
+import { Accordion, AccordionSet, Button, Card, Col, ErrorBoundary, KeyValue, Layout, MultiColumnList, Pane, Row } from '@folio/stripes/components';
+import { noop } from 'lodash';
+import { node } from 'prop-types';
+import { IItemRecordDetailPane, IWorkflowNodes } from '../../interfaces';
+
+import { t } from '../../utilities';
+
+export const ItemRecordDetailPane: React.FC<IItemRecordDetailPane> = ({ detailPaneSelect }) => {
+  const { selectedItem, showDetail, onRowClick, onClose } = detailPaneSelect;
+  const actionMenu = <Button bottomMargin0 buttonStyle="primary" onClick={noop}>{ t('button.actions') }</Button>;
+
+  if (!selectedItem) {
+    return <div>No item selected</div>;
+  }
+
+  if (!showDetail) {
+    return null;
+  }
+
+  return (
+    <ErrorBoundary>
+      <Pane
+        defaultWidth="fill"
+        dismissible
+        lastMenu={actionMenu}
+        onClose={onClose}
+        paneTitle={selectedItem?.name}
+      >
+        <AccordionSet>
+          <Accordion
+            label={<FormattedMessage id="ui-workflow.workflows.detail.item.label-name" />}
+            id={selectedItem?.id}
+          >
+            <Row>
+              <Col xs={12}>
+                <KeyValue label={<FormattedMessage id="ui-workflow.workflows.detail.item.name" />}>
+                  {
+                Boolean(selectedItem?.name) && (
+                  <>
+                    {selectedItem?.name}
+                  </>
+                )
+              }
```

Comment:
Please avoid doing it this way.

Instead, create a function or a type.

For exaxmple, consider the `<KeyValue ...>`.
A new component can be created that handles a boolean value, then you can do the following:

`<BooleanKeyValue label={..} value={selectedItem?.name}>`.

We may not have time to address this now, but I want to note this.